boltdesignsystem/bolt
# Refactor bullet styles, decrease UL spacing

## Jira

http://vjira2:8080/browse/BDS-2028

## Summary

Decreases the space between UL bullet and text, refactors how list spacing is set.

## Details

I refactored the list item spacing and the bullet position to be dynamic based on the value of a few local variables. As part of this work, I moved the UL bullet to align with the center of the OL number circle. This gives the appearance of the bullet being closer to the text.

## How to test

- Review refactored code
- OL: Verify there are no regression between [master](https://boltdesignsystem.com/pattern-lab/patterns/02-components-ol-15-ol-nested-items/02-components-ol-15-ol-nested-items.html) and [feature branch](https://boltdesignsystem-b2la3wgis.now.sh/pattern-lab/patterns/02-components-ol-15-ol-nested-items/02-components-ol-15-ol-nested-items.html)
- UL: Verify that the bullet is slightly indented on [feature branch](https://boltdesignsystem-b2la3wgis.now.sh/pattern-lab/patterns/02-components-ul-15-ul-nested-items/02-components-ul-15-ul-nested-items.html) compared to [master](https://boltdesignsystem.com/pattern-lab/patterns/02-components-ul-15-ul-nested-items/02-components-ul-15-ul-nested-items.html)
- Review above links cross-browser and verify the same
